pkg/planner/core/memtable_predicate_extractor.go

```diff
@@ -1812,7 +1812,7 @@ func (e *InfoSchemaTablesExtractor) Extract(ctx base.PlanContext,
 	predicates []expression.Expression,
 ) (remained []expression.Expression) {
 	var resultSet, resultSet1 set.StringSet
-	e.colNames = []string{"table_schema", "table_name"}
+	e.colNames = []string{"table_schema", "constraint_schema", "table_name", "constraint_name", "sequence_schema", "sequence_name", "partition_name", "schema_name", "index_name"}
 	e.ColPredicates = make(map[string]set.StringSet)
 	remained = predicates
 	for _, colName := range e.colNames {
```

Comment:
maybe batch all `colNames` into one call to reduce overhead. In the new code there are many colNames than before

Reply:
complicated to do so